browser/chat/chat_tab_helper.h

```diff
@@ -0,0 +1,77 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_CHAT_CHAT_TAB_HELPER_H_
+#define BRAVE_BROWSER_CHAT_CHAT_TAB_HELPER_H_
+
+#include <memory>
+#include <string>
+#include <vector>
+
+#include "brave/components/chat_ui/common/chat_ui.mojom.h"
+#include "content/public/browser/web_contents_observer.h"
+#include "content/public/browser/web_contents_user_data.h"
+
+namespace ui {
+class AXTree;
+}  // namespace ui
+
+using chat_ui::mojom::CharacterType;
+using chat_ui::mojom::ConversationTurn;
```

Comment:
Aliasing here could cause name collision. Could you move these to .cc file or to the class scope?